packages/fhir-converter/src/templates/cda/Resources/MedicationStatement.hbs

```diff
@@ -25,7 +25,7 @@
                 {
                     "doseQuantity":
                     {
-                        "value":{{medicationStatement.doseQuantity.value}},
+                        "value":"{{medicationStatement.doseQuantity.value}}",
```

Comment:
fyi Raf this patch would fix your error here:
https://metriport.slack.com/archives/C06CQC7GQG3/p1706031381120459?thread_ts=1706027906.239089&cid=C06CQC7GQG3